### PR TITLE
[TwigComponent] Fix link for the picture in README

### PR DESCRIPTION
**Pull request by mdoutreluingne**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| Tickets       | None
| License       | MIT

I fix the link for the picture alert-example in the README for TwigComponent.

### CODE REVIEW

**Review by kbond — APPROVED** [review]
Thanks, I did notice this the other day.

File: src/TwigComponent/README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ Done! Now render it wherever you want:
 
 Enjoy your new component!
 
-![Example of the AlertComponent](https://github.com/symfony/ux-twig-component/blob/main/alert-example.png?raw=true)
+![Example of the AlertComponent](https://github.com/symfony/ux/blob/2.x/src/TwigComponent/alert-example.png)
```
Comment (kbond) [inline src/TwigComponent/README.md]:
```suggestion
![Example of the AlertComponent](https://github.com/symfony/ux/blob/2.x/src/TwigComponent/alert-example.png?raw=true)
```

**Review by kbond — CHANGES_REQUESTED** [review]
Looks like github knows to embed the image but when I open in my ide, it's a broken link without the `?raw=true`.

**Comment by mdoutreluingne** [comment]
Interesting, I don't know, thank you for the remark, it is normally corrected

**Comment by kbond** [comment]
Good catch, thanks @mdoutreluingne.

**Pull request merged**